README.md

```diff
@@ -146,6 +146,50 @@ locations.forEach((location) => {
 ```
 **[⬆ back to top](#table-of-contents)**
 
+### Avoid Mental Mapping of iterator states
```

Comment:
Can you call this "Avoid Mental Mapping (part 2)" and call the one before it "Avoid Mental Mapping (part 1)".